### ISSUE TITLE
Overall view issues

### ISSUE THREAD

**Issue by 60trainhunter** [OpenRCT2/OpenRCT2#6151]
**OS:** [e.g. Windows 10]
**Version:** [e.g. 0.1.1]
**Commit/Build:** [e.g. 426e106]

1st overall is kinda zoomed too far out and you can barely see the ride it self. try merry go round the traditional one. 2nd then you want to move the ride or stall some where else in the park. as you can see overall view doesn't move with the rides or stalls. it's very easy to reproduce. i provided before and after shots of simple haunted house being placed and moved. 

![mot simple blank map 2017-08-10 14-54-50](https://user-images.githubusercontent.com/30908785/29187107-8f3ecdee-7ddc-11e7-99f4-ac688ef19694.png)
![mot simple blank map 2017-08-10 14-56-48](https://user-images.githubusercontent.com/30908785/29187114-9b6034c8-7ddc-11e7-935f-4e81d9cc5439.png)
![mot simple blank map 2017-08-10 14-57-10](https://user-images.githubusercontent.com/30908785/29187164-cb00a9b0-7ddc-11e7-99cd-e09f0f7e7466.png)

- [ no] Reproducible in RCT2 (vanilla)?
- [ yes] Multiplayer?

**Steps to reproduce:**
1. build any gentle or thrill rides and look at overall view.
2. then move that ride you placed some where else. look at overall view again.

**Screenshots / Video:**
 i gave some screenshots above

**Comment by rwjuk** [comment]
I was only able to reproduce part of the issue (the preview being zoomed out), not the issue after moving the ride. Could someone else have a go at replicating this?

**Comment by Chaosmeister** [comment]
## I was able to reproduce both issues:

* 1. on the merry-go-round on SixFlags Belgium, but only if I click on the sign of the queue line.
Clickin on the ride, entry and exit all result in a correct (near) zoom **as long as it's entry hasn't been moved**. After moving the Entry, the zoom on clicking the merry go round is incorrect as well **as long, as it hasn't been tested or opened**.

* 2. requires a little bit more effort and exact workflow:
Close the merry go round and remove it's entry and exit by rightclicking each and then rightclicking the merry go round. After that remove the ride itself with the bulldozer button and rebuild it anywhere else in the same step. Finally place the entry and exit so the construction window is closed. Now the ride view will zoom to the old position.

![image](https://user-images.githubusercontent.com/13346344/29498778-1eb49d54-8603-11e7-90a9-3f1ca51f3b8b.png)

**Comment by Broxzier** [comment]
Some things I've observed:
- The "Overall view" is _not_ always zoomed out (motion simulator, haunted house are not, all other rides that I tester are)
- The "Station" view is always zoomed in
- Clicking on the attraction before it has been opened, goes to "Overall view".
- Clicking on it after having opened the attraction, it goes to "Wheel 1", "Building 1", etc... instead of "Overall view"
     - (as if the building has appeared, much like how vehicles appear once opening a tracked ride. Before opening, these options are greyed out, and after double-closing they disappear again. [Turning on "See-through Rides" confirms this](https://i.imgur.com/cm8nxNi.gif).)
- Clicking on the entrance or queue sign goes to "Station 1" view
- Clicking on the brown floor underneath the ride opens "Overall view"
- Clicking on customized floors opens "Station 1" view (the ones that observation towers have for example)

The only exception to these rules I found are spiral slides. These don't have a "Building 1", and always seem to open the "overall view" instead, even when clicking on the entrance. Perhaps it's because they don't have a test option, much like stalls. This also applies when guests are inside.
And of course mazes, that only have an "Overall view".

I think it wouldn't hurt to change the default zoom level for all flat rides to lowest zoom level.
